### ISSUE TITLE
[Bug]: View, Translation and Route not found warning event if it exists.

### ISSUE THREAD

**Issue by AHMED-GAMAL-AG** [laravel/vs-code-extension#132]
### Extension Version

0.1.14

### PHP Binary

Herd

### Operating System

Windows

### What happened?

I get a warning that View, Translation, Route, etc., are not found, even if it exists.

![image](https://github.com/user-attachments/assets/65816985-6e25-43ae-99df-e2bafea8af00)

**Comment by joetannenbaum** [comment]
Yes, we are still porting the full extension functionality over to Blade, it's on the roadmap:

https://github.com/laravel/vs-code-extension?tab=readme-ov-file#on-the-roadmap

**Comment by n1crack** [comment]
You should add support for Gates 

Gate::authorize("update", Post::class)

Should go to PostPolicy@update

**Comment by TitasGailius** [comment]
Thanks for reporting this. We recently released multiple improvements to how Blade and Inertia views are discovered, so I’m hoping this is now fixed. Feel free to reopen the issue if you’re still running into problems.
